src/pages/member/NewMemberRegister.js

```diff
@@ -34,13 +35,13 @@ const NewMemberRegisterFormContainer = (props) => {
   const classes = useStyles();
 
   const history = useHistory();
-  const useQuery = () => { return new URLSearchParams(useLocation().search) };
```

Comment:
I worry about people refreshing on NewMemberRegister and losing their redux store, maybe query params is a better solution?

Reply:
So far it doesn't seem to be a problem because the top of `route.js` calls `if(!this.props.user) this.getAuthenticatedUser();`, so when the user is logged in, they always have a `user` object in redux store... Do you think using query params is more robust anyways?

Reply:
In that case redux is good